addons/new-osio-addon/src/main/java/io/fabric8/launcher/osio/OsioMissionControl.java

```diff
@@ -65,39 +59,37 @@ public Boom launch(Projectile genericProjectile) throws IllegalArgumentException
         OsioProjectile projectile = (OsioProjectile) genericProjectile;
         GitRepository repository = gitSteps.createRepository(projectile);
 
+        executeCommonSteps(projectile, repository);
+
+        gitSteps.pushToGitRepository(projectile, repository);
+
+        openShiftSteps.triggerBuild(projectile);
+        return ImmutableBoom.builder()
+                .createdRepository(repository)
+                .createdProject(ImmutableOpenShiftProject.builder().name(projectile.getOpenShiftProjectName()).build())
```

Comment:
Have you tested if this code works? The builder as it is defined right now requires you to specify a the consoleOverviewUrl. Unless you add a @Nullable to it